src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue8788.cs

```diff
@@ -0,0 +1,24 @@
+﻿#if ANDROID
```

Comment:
Can we run this test on iOS and Windows also? 

I realize the bug only talks about android but since this code is relevant on all platforms, it's ideal to have a test that runs on all three